discovery-provider/src/queries/get_users_cnode.py

```diff
@@ -70,6 +70,7 @@ def get_users_cnode(cnode_endpoint_string, replica_type=ReplicaType.PRIMARY):
             t.secondary1 is not NULL;
             """
         )
+        session.info['src'] = get_users_cnode.__name__
```

Comment:
can we make this a decorator on the function to add this?

Reply:
Not sure how to accomplish this with a decorator but applied it into every session instead.